user/migrations/0001_initial.py

```diff
@@ -126,7 +110,7 @@
                 "abstract": False,
             },
             managers=[
-                ("objects", django.contrib.auth.models.UserManager()),
+                ("objects", user.models.UserManager()),
```

Comment:
The `UserManager` is correctly assigned as the manager for the `User` model, ensuring custom user management logic is applied.